src/keeshare/CMakeLists.txt

```diff
@@ -1,4 +1,18 @@
 if(WITH_XC_KEESHARE)
+    set(WITH_XC_KEESHARE_INSECURE ON PARENT_SCOPE)
+
+    if(WITH_XC_KEESHARE_SECURE)
+        find_package(QuaZip REQUIRED)
+    else()
```

Comment:
Why not simply scrap this option and let `find_package()` try to find it without `REQUIRED` in the first place? That's basically what it's been designed for. Then if `QUAZIP_FOUND` is set, we can add the necessary defines and if not, we issue a CMake warning. That way we don't need to bother users with `XC_KEESHARE_SECURE` at all.

Reply:
Good idea, and now that quazip is fixed for macOS that makes even more sense.